ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json

```diff
@@ -339,112 +339,113 @@
                 },
                 "TissueAndMedicalRecords": {
-                    "Question": "[es]7. If my tissue and medical records are analyzed, will I receive any information back from this study?",
-                    "Text": "[es]Because our sequencing tests are performed in a research lab and not a clinical lab, we are not currently permitted to return personal genetic data to you. However, we will regularly update you with key discoveries that we make about this disease, overall results, and progress made through this research."
+                    "Question": "7. Si mi tejido de tumor y datos clínicos son analizados, ¿recibiré la información de mis estudios genéticos?",
+                    "Text": "Nuestros estudios de secuenciación genómica son realizados en un laboratorio de investigación y no en un laboratorio clínico, por esta razón no tenemos permitido regresarle sus datos genéticos personales. Se le mantendrá informada/o sobre información nueva e importante sobre esta enfermedad, resultados sobre el proyecto y el progreso de la investigación."
                 },
                 "Profit": {
-                    "Question": "[es]8. Is this research being conducted for profit?",
-                    "Text": "[es]No. This project is a not-for-profit, academic research endeavor. Our mission is to generate, aggregate, and freely share data with the research community in a way that protects your personal information in order to fuel discoveries."
+                    "Question": "8. ¿Esta investigación se realiza con fines de lucro?",
+                    "Text": "No. Este proyecto es un esfuerzo sin fines de lucro. Nuestra misión es generar, compilar y compartir libremente datos con la comunidad de investigación de forma que se proteja su información personal para acelerar descubrimientos científicos en cáncer."
                 },
                 "InfoForPhysicians": {
-                    "Question": "[es]9. Is there information that I can bring to my doctor so that I can discuss this study with them?",
-                    "Text": "[es]Yes, you can print out or use your mobile device to share the document linked here:",
-                    "Link": "[es]Metastatic Breast Cancer Project For Physicians"
+                    "Question": "9. ¿Hay alguna información que pueda proporcionar a mi médico y/u hospital para poder hablar sobre este estudio con ellos?",
+                    "Text": "Sí, puede imprimir o usar su dispositivo móvil para compartir este documento:",
+                    "Link": "The Metastatic Breast Cancer Project para Médicos"
```

Comment:
`El Metastatic Breast Cancer proyecto para Médicos`
seems to be more proper

Reply:
I replaced The with El